apps/provider-proxy/src/app.ts

```diff
@@ -0,0 +1,33 @@
+import cors from "cors";
+import express from "express";
+
+import { getAppStatus } from "./routes/getAppStatus";
+import { proxyProviderRequest } from "./routes/proxyProviderRequest";
+
+export const app = express();
+
+const whitelist = [
```

Comment:
It would be nice to receive those as env vars, it's something we've postponed for a while, but doesn't take too much effort to implement.